### PR TITLE
chore: suppress issues

### PR DESCRIPTION
**Pull request by pbabz**
Issue #, if available:
GALI-1866

Description of changes:
Suppress CFN nag errors and warnings.

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/SWBv2.postman_collection.json
Comment (sevu1) [inline solutions/swb-reference/SWBv2.postman_collection.json:39]: What was the change in this file except formatting?

Comment (SanketD92) [reply solutions/swb-reference/SWBv2.postman_collection.json:39]: No other changes

**Review by SanketD92 — APPROVED** [review]

**Review by sevu1 — APPROVED** [review]

**Review by insignias — APPROVED** [review]

**Comment by insignias** [comment]
Approving this as I was notified the findings will be addressed in another PR. The changes in this code just ignores the cfn nag findings and hence, when cfn nag pipeline workflow is activated it will not fail

**Review by insignias — APPROVED** [review]

**Comment by SanketD92** [comment]
Changing base branch to `develop-pipeline` to unblock solutions pipeline integration work. Fixes to the TODO comments/ignored CFN nag warnings will be fixed before merging with develop.

File: solutions/swb-reference/src/SWBStack.ts
Comment (SanketD92) [reply solutions/swb-reference/src/SWBStack.ts:373]: Changing branch to develop-pipeline

**Pull request merged**
